patches/kaggle_secrets.py

```diff
@@ -12,7 +10,7 @@
 
 _KAGGLE_DEFAULT_URL_BASE = "https://www.kaggle.com"
 _KAGGLE_URL_BASE_ENV_VAR_NAME = "KAGGLE_URL_BASE"
-_KAGGLE_USER_SECRETS_TOKEN_ENV_VAR_NAME = "KAGGLE_USER_SECRETS_TOKEN_KEY"
+_KAGGLE_USER_SECRETS_TOKEN_ENV_VAR_NAME = "KAGGLE_USER_SECRETS_TOKEN"
```

Comment:
What's the plan for this env var rename?

We set it at worker startup here: https://github.com/Kaggle/Kaggle.SDS.Worker/blob/ci/kernels/worker/init/setup.sh#L147

Reply:
Right, I had made a mistake originally calling TOKEN_KEY here, what we really want is the TOKEN itself: https://github.com/Kaggle/Kaggle.SDS.Worker/blob/7e5c918306f1424739dcb104b39e2449980320fd/kernels/worker/src/kernel_run_container_commands.py#L65. The token key is used to generate the token in the user_secrets_token_gen.py